sympy/integrals/rubi/utility_function.py

```diff
@@ -258,7 +261,7 @@ def IntegersQ(*var):
     return all(IntegerQ(i) for i in var)
 
 def ComplexNumberQ(*var):
-    """
+    r"""
```

Comment:
Why use `r` here?
Python documentation says:
```
Use r"""raw triple double quotes""" if you use any backslashes in your
docstrings.
```
See https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring